tests/drivers/test_forecast.py

```diff
@@ -0,0 +1,52 @@
+#pylint: disable=invalid-name, missing-module-docstring, missing-function-docstring
+#pylint: disable=unused-variable
+
+import os
+import shutil
+import glob
+import pytest
+
+from uwtools.drivers.driver import Driver
+from uwtools.drivers.forecast import FV3Forecast
+
+
+def test_create_directory_structure():
+    """Tests create_directory_structure method given a directory."""
+    forecast_obj = FV3Forecast(Driver)
+    uwtools_file_base = os.path.join(os.path.dirname(__file__))
+    run_directory = os.path.join(uwtools_file_base, "expt_dir")
```

Comment:
Does it make sense to use a named temporary directory for this path so that it's less likely to have collisions with other tests?

It also means we wouldn't have to remove anything that may already be existing, as in the next two lines.